sentinel-core/src/core/config/base.rs

```diff
@@ -244,3 +244,11 @@ pub fn metric_stat_sample_count() -> u32 {
         .try_with(|c| c.borrow().config.stat.sample_count)
         .unwrap()
 }
+
+#[inline]
+pub fn get_default_log_dir() -> String {
+    match dirs::home_dir() {
+        Some(path) => path.join(LOG_METRICS_DIR).to_string_lossy().to_string(),
```

Comment:
I think the default behavior should be the same as the [Sentinel](https://github.com/alibaba/Sentinel) and [sentinel-golang](https://github.com/alibaba/sentinel-golang). If you need a different log directory, you can set it with a configuration entity or a configuration file, see these examples:

https://github.com/sentinel-group/sentinel-rust/blob/main/examples/config/from_yaml/from_yaml.rs
https://github.com/sentinel-group/sentinel-rust/blob/main/testdata/config/sentinel.yaml
or 
https://github.com/sentinel-group/sentinel-rust/blob/main/examples/config/from_entity/from_entity.rs

Reply:
@sczyh30 @LearningGp If you think this minor difference is acceptable, I have no other concerns.

Reply:
@Forsworns But, it seems that Golang SDK use `homedir` as its default path, please see this link: https://github.com/alibaba/sentinel-golang/blob/08071855bc67423777353c6e02f1390b419172b1/core/config/config.go#L178
I'm not familiar with sentinel source codes, please confirm, thanks!

Reply:
@jjeffcaii I see, I forget why this is different from the sentinel-golang, it should not have been.